### ISSUE TITLE
Running example on Windows fails with attempt to pass additional trust anchors.

### ISSUE THREAD

**Issue by aam** [aam/grpc_cronet#2]
```
PS C:\src\grpc_cronet\example> C:\src\f\flutter\bin\flutter.bat run -v
...
[   +6 ms] flutter: bicronet_grpc: CronetGrpcClientConnection dispatchCall Instance of 'ClientCall<HelloRequest, HelloReply>'
[   +2 ms] [0713/143615.992:FATAL:multi_threaded_cert_verifier.cc(257)] Attempted to set a CertVerifier::Config with additional trust
anchors, but |verify_proc_| does not support additional trust anchors.
[        ] Backtrace:
[        ]      bidirectional_stream_cancel [0x00007FFA518C6282+1380610]
[        ]      bidirectional_stream_cancel [0x00007FFA51798C32+146098]
[        ]      bidirectional_stream_cancel [0x00007FFA517A73D5+205397]
[        ]      bidirectional_stream_cancel [0x00007FFA5193FA8B+1878283]
[        ]      bidirectional_stream_cancel [0x00007FFA51942446+1888966]
[        ]      bidirectional_stream_cancel [0x00007FFA5194316A+1892330]
[        ]      bidirectional_stream_engine_destroy [0x00007FFA51699284+2372]
[        ]      Cronet_RequestFinishedInfo_finished_reason_get [0x00007FFA5177184E+786446]
[        ]      Cronet_RequestFinishedInfo_finished_reason_get [0x00007FFA51773F7C+796476]
[        ]      bidirectional_stream_cancel [0x00007FFA518D2710+1430928]
[        ]      GetHandleVerifier [0x00007FFA51A43819+147417]
[        ]      GetHandleVerifier [0x00007FFA51A42B52+144146]
[        ]      GetHandleVerifier [0x00007FFA51A3F4AB+130155]
[        ]      GetHandleVerifier [0x00007FFA51A3CA95+119381]
[        ]      GetHandleVerifier [0x00007FFA51A44DEB+153003]
[        ]      bidirectional_stream_cancel [0x00007FFA518F0927+1554343]
[        ]      bidirectional_stream_cancel [0x00007FFA517CF5A0+369696]
[        ]      bidirectional_stream_cancel [0x00007FFA517CF8F0+370544]
[        ]      bidirectional_stream_cancel [0x00007FFA517E2119+446361]
[        ]      BaseThreadInitThunk [0x00007FFAB3DE7034+20]
[        ]      RtlUserThreadStart [0x00007FFAB3F22651+33]
```

**Comment by aam** [comment]
additional trusted anchors are only supported on linux it seems, different mechanism to run test needs to be used on windows.

**Comment by aam** [comment]
Fixed in 4e95d9d1b64eeaa178b1161180dba8a356a0731c
